trunk/includes/pardot-settings-class.php

```diff
@@ -130,50 +134,34 @@ function __construct() {
 		/**
 		 * Configure the settings page for the Pardot Plugin if we are currently on the settings page.
 		 */
-		add_action( 'admin_init', array( $this, 'admin_init' ) );
+		add_action('admin_init', [$this, 'admin_init']);
 
 		/**
 		 * Configure the admin menu that points to this settings page for the Pardot Plugin.
 		 */
-		add_action( 'admin_menu', array( $this, 'admin_menu' ) );
+		add_action('admin_menu', [$this, 'admin_menu']);
 
 		/**
 		 * Present an admin message telling the user to configure the plugin if there are not yet any credentials.
 		 * This gets displayed at the top of an admin page.
 		 */
-		add_action( 'admin_notices', array( $this, 'admin_notices' ) );
+		add_action('admin_notices', [$this, 'admin_notices']);
 
 		/**
 		 * Enqueue JS for Chosen on Widgets Screen
 		 */
-		add_action( 'admin_enqueue_scripts', array( $this, 'pardot_chosen_enqueue' ) );
-
-
-        /**
-         * Because a crypto key is *REQUIRED* we're going to check to see if a pardot crypto key exists
-         * in the settings table, and if it doesn't we're going to create one for use. We only ever want to do this
-         * one time so it's not going to be continuing logic and it won't be stored with other pardot settings
-         */
-        if (get_option('pardot_crypto_key', NULL) === NULL) {
-            $crypto = new PardotCrypto();
-            $crypto->set_key();
-        }
-
-
-        /**
-         * And we're going to check here to see if the setting for pardot already exists in the database for their password...
-         * if it does then we need to determine if the password setting is correctly encrypted or needs to be re-encrypted
-         */
-        $optstring = get_option('pardot_settings', NULL);
-        if (isset($optstring['password']) && $optstring['password'] != NULL)
-        {
-            if ((substr($optstring['password'], 0, 6) !== "NACL::") and
-                (substr($optstring['password'], 0, 6) !== "OGCM::") and
-                (substr($optstring['password'], 0, 6) !== "OETM::"))
-            {
-                self::upgrade_old_password($optstring['password']);
-            }
-        }
```

Comment:
Logic used for updating the password encryption.  Since we don't use Pardot auth anymore, we don't have a locally stored password, so we don't need this anymore.